### PR TITLE
Update for project restructuring

### PR DESCRIPTION
**Pull request by scottslowe**
This PR updates the repository for the project restructuring post. This includes removing all the code for the base infrastructure and Kubernetes cluster (these are moved into separate projects), and using a StackReference to obtain the necessary information to deploy the application. All documentation is updated as well.

Once this PR is merged, then a tag can be pushed for the third blog post.

### CODE REVIEW

File: infra/index.ts [hunk 1]
```diff
@@ -1312,11 +1290,15 @@ const rabbitmqDeployment = new k8s.apps.v1.Deployment("rabbitmq-deployment", {
 
 const uiImage = new awsx.ecr.Image("ui-image", {
     repositoryUrl: repository.url,
-    dockerfile: "../images/java17/Dockerfile",
-    path: "../src/ui",
+    dockerfile: "/Users/slowe/Work/Code/Repos/zephyr-app/images/java17/Dockerfile",
+    path: "/Users/slowe/Work/Code/Repos/zephyr-app/src/ui",
```
Comment (cnunciato) [inline infra/index.ts:1294]: Should these be something else?

**Review by cnunciato — APPROVED** [review]
Couple questions, nothing need block.

File: infra/index.ts (hunk 1 above)
Comment (scottslowe) [reply infra/index.ts:1294]: Yes, I'll need to fix this. Thanks for catching it!

File: infra/index.ts [hunk 2]
```diff
@@ -1,40 +1,19 @@
...
-const vpcNetworkCidr = config.get("vpcNetworkCidr") || "10.0.0.0/16";
-
-// Create a new VPC
-const eksVpc = new awsx.ec2.Vpc("eks-vpc", {
-    enableDnsHostnames: true,
-    cidrBlock: vpcNetworkCidr,
-});
+const eksOrgName = config.get("eksOrgName");
+const eksProjName = config.get("eksProjName");
+const eksStackName = config.get("eksStackName");
```
[the remark this answers is not in the document]
Comment (scottslowe) [reply infra/index.ts:9]: I'll change them from `eks` to `k8s`, and I'll make them `require`s as well. At this stage in the game (when we need to get this post published), I'm not terribly concerned about platform vs. k8s. We can show Zephyr using `pulumi stack rename` later in the series. :-)

File: infra/index.ts (hunk 1 above)
Comment (scottslowe) [reply infra/index.ts]: Fixed in 624ec89

File: infra/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (scottslowe) [reply infra/index.ts]: Fixed in 624ec89

**Pull request merged**